pkg/kritis/metadata/containeranalysis/cache.go

```diff
@@ -28,7 +28,7 @@ import (
 type Cache struct {
 	client metadata.ReadWriteClient
 	vuln   map[string][]metadata.Vulnerability
-	att    map[string][]metadata.PGPAttestation
+	att    map[string][]metadata.RawAttestation
```

Comment:
atts? 
cc @jialur for go formality.

Reply:
atts is fine with me.

Reply:
@jialur thanks.
Cool. I'm also fine fixing these in a separate PR (potentially addressing other similar issues, e.g., s/vuln/vulns/g)

Reply:
Done. We can do the others in another PR